package.json

```diff
@@ -29,11 +29,12 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@dvsa/cvs-type-definitions": "7.1.0",
     "@aws-sdk/client-lambda": "^3.549.0",
     "@aws-sdk/client-s3": "^3.550.0",
     "@aws-sdk/client-secrets-manager": "^3.549.0",
     "@aws-sdk/util-dynamodb": "^3.592.0",
-    "@dvsa/cvs-microservice-common": "^0.9.8",
+    "@dvsa/cvs-microservice-common": "^1.1.0",
```

Comment:
Is cvs-microservice-common actually used in this service now? It used to be used to processRecord, but that was removed with the Notify Error Handling fix ticket. I know from CB2-12738 we were able to remove it.